<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ethernet Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>321</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="uartButtonLayout">
     <item>
      <widget class="QComboBox" name="comboBoxUartBaudrate"/>
     </item>
     <item>
      <widget class="QPushButton" name="connectButtonUart">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ethernetButtonLayout">
     <item>
      <widget class="QComboBox" name="comboBoxEthernetConnectionTypes"/>
     </item>
     <item>
      <widget class="QPushButton" name="connectButtonEthernet">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>671</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ethernetBrowserLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextBrowser" name="textBrowserEthernet"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="ethernetSendButton">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowserUart"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
